### PR TITLE
747 path cli refactoring

### PR DESCRIPTION
**Pull request by doori**
- rename container -> datastore
- add unittests

### CODE REVIEW

**Review by druvpatel — COMMENTED** [review]
looks great! :)

File: data_processing/common/DataStore.py [hunk 1]
```diff
@@ -206,24 +206,24 @@ def setContainer(self, container_id):
...
         # Attach
         cohort = Node("cohort", self._namespace_id)
-        if not len(self._conn.query(f""" MATCH (co:{cohort.get_match_str()}) MATCH (container) WHERE id(container) = {self._container_id} MERGE (co)-[:INCLUDE]->(container) RETURN co,container """ ))==1: 
-            self.logger.warning ( "Cannot attach, tried [%s]", f""" MATCH (co:{cohort.get_match_str()}) MATCH (container) WHERE id(container) = {self._container_id} MERGE (co)-[:INCLUDE]->(container) RETURN co,container """)
+        if not len(self._conn.query(f""" MATCH (co:{cohort.get_match_str()}) MATCH (container) WHERE id(container) = {self._datastore_id} MERGE (co)-[:INCLUDE]->(container) RETURN co,container """)) == 1:
+            self.logger.warning ( "Cannot attach, tried [%s]", f""" MATCH (co:{cohort.get_match_str()}) MATCH (container) WHERE id(container) = {self._datastore_id} MERGE (co)-[:INCLUDE]->(container) RETURN co,container """)
             return self
 
         # Let us know attaching was a success! :)
-        self.logger = init_logger(f'logs/container-{self.address}.log')
```
Comment (aauker) [inline data_processing/common/DataStore.py:218]: I've been meaning to raise this issue, to improve logging-- perhaps it's time to fetch the actual log dir?  Opened https://github.com/msk-mind/backlog/issues/776

File: data_processing/pathology/cli/visualize_tile_labels.py [hunk 2]
```diff
@@ -68,31 +66,45 @@ def visualize_tile_labels_with_container(cohort_id: str, container_id: str, meth
...
+
+                # build viz
+                result = subprocess.run(["python3","-m","data_processing.pathology.cli.dsa.dsa_viz",
+                                         "-c", f"{tmpdir}/dsa_config.json", "heatmap",
+                                         "-d", f"{tmpdir}/model_inference_config.json"],
+                                        stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True)
+                print(result.returncode, result.stdout, result.stderr)
+
+                # push results to DSA
+                subprocess.run(["python3","-m","data_processing.pathology.cli.dsa.dsa_upload",
```
Comment (aauker) [inline data_processing/pathology/cli/visualize_tile_labels.py:98]: Brilliant--- it'll be quite nice to have this working in full

**Review by aauker — APPROVED** [review]
Overall, very comprehensive PR, and unit and integration tests are especially great. 

I think the `method_data.get("env", "data")` use is in the right direction, but there might be a more elegant way to do this--- the "dev environment" could perhaps be part of the "namespace", e.g. dev/lung-ct-radiomics or prod/ovarian-classifier-tiles or something along those lines.  

Approved!

**Pull request merged**